### PR TITLE
Fix fontloader polyfill by comparing against one default font at a time.

### LINKED ISSUE #6219: amp-font does not work in Microsoft Edge
We are having trouble loading webfonts with the <amp-font> component. We threw together a quick example. Are we doing something wrong? We see some errors in the console, might be related.

## How do we reproduce the issue?

1. Open the Edge browser
2. Navigate to https://output.jsbin.com/tofusinehe/
3. See that "fancy font" is not properly styled 

## What browsers are affected?

Edge 38.14393.0.0

## Which AMP version is affected?

AMP 1478301221318
I don't think this is a new issue.

### PR DESCRIPTION
**Pull request by cvializ**
Fixes #6219

The polyfill in `fontloader.js` wasn't working because `compareMeasurements_` checked if the test div's font was different than both a `sans-serif` and a `serif` font at the same time. `compareMeasurements_` works by comparing the dimensions of two divs containing the same special text string. If the two divs have different fonts, they should have different dimensions. The code assumes that if the dimensions are different, the font must have loaded.

Since the test div was given `font-family: "Comic AMP", sans-serif, serif`, it would be rendered with a `sans-serif` font before `"Comic AMP"` loaded or failed to load. Since `compareMeasurements_` compared against `serif` *and* `sans-serif`, and the test div rendered with `sans-serif`, the dimensions of the test div would be different than the dimensions of the `serif` div. This caused `amp-font` to behave as if the font had loaded when it really hadn't.

tl;dr Fixed the bug by giving the test div only one fallback font-family at a time.

### CODE REVIEW

**Comment by cvializ** [comment]
This is ready to merge any time.
